src/codec/id.rs

```diff
@@ -907,7 +923,7 @@ impl From<AVCodecID> for Id {
 			AV_CODEC_ID_HDMV_TEXT_SUBTITLE => Id::HDMV_TEXT_SUBTITLE,
 
 			/* other specific kind of codecs (generally used for attachments) */
-			AV_CODEC_ID_TTF       => Id::TTF,
+			AVCodecID_AV_CODEC_ID_TTF       => Id::TTF,
```

Comment:
No I meant, keeping at as an enum, but all the helper entries like `AV_CODEC_ID_FIRST_UNKNOWN` would become `pub const AV_CODEC_ID_FIRST_UNKNOWN: AVCodecID = AV_CODEC_ID_TTF`.